Core/Main/elxElastixFilter.hxx

```diff
@@ -275,7 +275,7 @@ ElastixFilter< TFixedImage, TMovingImage >
   } // End loop over registrations
 
   // Save result image
-  if( resultImageContainer.IsNotNull() && resultImageContainer->Size() > 0 )
+  if( resultImageContainer.IsNotNull() && resultImageContainer->Size() > 0 && resultImageContainer->ElementAt( 0 ).IsNotNull() )
```

Comment:
Did any problem occur before you added the extra `resultImageContainer->ElementAt( 0 ).IsNotNull()` check?

Reply:
No this is just an extra check to be absolutely certain that the object is not null. There was no specific incident that made this necessary.

Reply:
OK!